backend/control_center/snmp_monitoring/migrations/0001_initial.py

```
@@ -0,0 +1,123 @@
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('network_device', '0004_alter_networkdevice_operating_system'),
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)